spec/features/create_goobi_image_spec.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+require 'druid-tools'
```

Comment:
Is this dependency still used below or can we nix it?

Reply:
good catch, it was copypasta and I removed it